babel.config.js

```diff
@@ -0,0 +1,24 @@
+module.exports = function (api) {
+  const isTest = api.env('test');
+  const presets = isTest ? [
+    [
+      "@babel/preset-env",
+      {
+        "targets": {"node":"current"}
```

Comment:
I wonder what the babel output is with `"targets": {"node":"current"}` and how it is different from the source.

Reply:
I also don't know, but this is documented as required in jest, and without it there is some strange environment error.